raiden/ui/cli.py

```diff
@@ -204,23 +205,35 @@ def app(address,
 
         address = addresses[idx]
 
-    unlock_tries = 3
-    while True:
+    password = os.environ.get('ETH_PASSWORD')
```

Comment:
Yes `--password-file` as a secondary method of providing the password for automated testing environments could be a compromise.